azurerm/resource_arm_servicebus_namespace.go

```diff
@@ -134,7 +139,7 @@ func resourceArmServiceBusNamespaceRead(d *schema.ResourceData, meta interface{}
 
 	resp, err := namespaceClient.Get(resGroup, name)
 	if err != nil {
-		return fmt.Errorf("Error making Read request on Azure ServiceBus Namespace %s: %+v", name, err)
+		return fmt.Errorf("Error making Read request on Azure ServiceBus Namespace '%s': %+v", name, err)
```

Comment:
`%q` would add double quotes around the string for you + escaped it properly here.